### PR TITLE
:bug: source mixpanel: hardcoded 'standard' properties for engage stream

### LINKED ISSUE #7124: source-mixpanel sync error (datatypes)
## Enviroment
- **Airbyte version**:  0.30.20-alpha
- **OS Version / Instance**: macOS
- **Deployment**: Docker compose
- **Source Connector and version**: Mixpanel 0.1.1
- **Destination Connector and version**: Postgres 0.3.10
- **Severity**: High
- **Step where error happened**: Sync job

## Current Behavior
Mixpanel synchronization failed at the Basic normalization step. Streams `cohort_members` and `engage` end with errors e.g. `invalid input syntax for type double precision: "89.0.4389.114"`.

## Expected Behavior
All streams should be synced successfully.

## Logs

[logs-62-0.txt](https://github.com/airbytehq/airbyte/files/7365409/logs-62-0.txt)

## Steps to Reproduce
1. Create MixPanel source (with cohort_members and engage data)
2. Connect to Postgres
3. Start sync

It seems that some fields in the catalog have different data types in Mixpanel data. e.g. `browser_version`. There might be more fields types differences. It should be checked.

### PR DESCRIPTION
**Pull request by midavadim**
:bug: source mixpanel: hardcoded 'standard' properties for engage stream

## What
Shortly about the problem.
Mixpanel 'engage' stream has dynamic schema. There are some 'standard' fields but users can create custom fields as well.
I have tried to create such 'custom' fields but there is no way to specify type for such custom field. I tried to create fields with different values: abc or 1. Such attrs are reported as string for me. I was not able to find any way to change this.
Customer has a problem with 'browser_version' field, which is a 'standard' field. It is reported as:
    'number' - for customer
    'string' - for our account.
Customer and I have similar values in this field, like "86.3.112.2", which is definitely a string.
So it is not clear what to do in this situation.

## How
solution is: we can hard-code types for known 'standard' fields, but rely on API for another 'custom' fields. What do you think?

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> New Connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector added to connector index like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
   
</p>
</details>

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</p>
</details>

<details><summary> <strong> Connector Generator </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: docs/integrations/sources/mixpanel.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Select the correct region \(EU or US\) for your Mixpanel project. See detail [he
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| `0.1.2` | 2021-10-30 | [6075](https://github.com/airbytehq/airbyte/issues/6075) | Hardcoded 'standard' properties for engage stream |
```
Comment (bazarnov) [inline docs/integrations/sources/mixpanel.md:59]: Let's change `hardcoded` to `Implemented`

**Review by bazarnov — CHANGES_REQUESTED** [review]
LGTM. Let's change the note of release inside of the change-log.

**Review by bazarnov — APPROVED** [review]

**Review by antixar — APPROVED** [review]

File: docs/integrations/sources/mixpanel.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ Select the correct region \(EU or US\) for your Mixpanel project. See detail [he
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| `0.1.2` | 2021-10-30 | [7505](https://github.com/airbytehq/airbyte/issues/6075) | Implemented 'standard' properties for engage stream |
```
Comment (sherifnada) [inline docs/integrations/sources/mixpanel.md:59]:
```suggestion
| `0.1.2` | 2021-10-30 | [7505](https://github.com/airbytehq/airbyte/issues/6075) | Guarantee that standard and custom mixpanel properties in the `Engage` stream are written as strings |
```

**Review by sherifnada — APPROVED** [review]

**Pull request merged**